src/main/java/com/epam/reportportal/auth/OAuthSuccessHandler.java

```diff
@@ -65,19 +65,17 @@
 
 	@Override
 	protected void handle(HttpServletRequest request, HttpServletResponse response, Authentication authentication) throws IOException {
-		String login = ofNullable(((OAuth2User) authentication.getPrincipal()).getAttributes().get(LOGIN_ATTRIBUTE)).map(String::valueOf)
+		String login = ofNullable((OAuth2Authentication) authentication).map(auth -> String.valueOf(auth.getPrincipal()))
 				.orElseThrow(() -> new ReportPortalException(ErrorType.ACCESS_DENIED,
-						Suppliers.formattedSupplier("Attribute - {} was not provided.", LOGIN_ATTRIBUTE).get()
+						Suppliers.formattedSupplier("{} was not provided.", LOGIN_ATTRIBUTE).get()
```

Comment:
It seems we dont need login_attribute constatnt anymore

Reply:
removed